README.md

```diff
@@ -38,6 +38,21 @@ This tutorial is based on AIS data from 11/20/2019 to 11/27/2019 on which we hav
  - 220051000
```

Comment:
in Line 35
I would say `This tutorial is based on AIS data emitted from 11/20/2019 to 11/27/2019. We extracted boats positions having the following MMSI`